### PR TITLE
Completed forum

### PR DESCRIPTION
**Pull request by margawron**
Fully functional forum, has really crude css styling.

### CODE REVIEW

File: PhotoPlusBackend/photoplus/.idea/intellij-javadocs-4.0.1.xml [hunk 1]
```diff
@@ -0,0 +1,204 @@
+<?xml version="1.0" encoding="UTF-8"?>
```
Comment (syphzur) [inline PhotoPlusBackend/photoplus/.idea/intellij-javadocs-4.0.1.xml]: why

Comment (syphzur) [reply PhotoPlusBackend/photoplus/.idea/intellij-javadocs-4.0.1.xml]: pls dont to that

**Review by syphzur — CHANGES_REQUESTED** [review]
please delete xml file from pull request

File: PhotoPlusBackend/photoplus/.idea/intellij-javadocs-4.0.1.xml [hunk 2]
```diff
@@ -0,0 +1,204 @@
...
+<project version="4">
+  <component name="JavaDocConfiguration">
```
Comment (dsuchy) [inline PhotoPlusBackend/photoplus/.idea/intellij-javadocs-4.0.1.xml]: oh no

**Review by dsuchy — CHANGES_REQUESTED** [review]

**Review by dsuchy — CHANGES_REQUESTED** [review]
W H Y ? ?

W H Y ? ?

File: PhotoPlusBackend/photoplus/.idea/intellij-javadocs-4.0.1.xml (hunk 1 above)
Comment (margawron) [reply PhotoPlusBackend/photoplus/.idea/intellij-javadocs-4.0.1.xml]: guess thats intelij making this ...

File: PhotoPlusBackend/photoplus/.idea/intellij-javadocs-4.0.1.xml (hunk 2 above)
Comment (margawron) [reply PhotoPlusBackend/photoplus/.idea/intellij-javadocs-4.0.1.xml]: Fixed in e07995e

File: PhotoPlusBackend/photoplus/.idea/intellij-javadocs-4.0.1.xml (hunk 1 above)
Comment (margawron) [reply PhotoPlusBackend/photoplus/.idea/intellij-javadocs-4.0.1.xml]: Fixed in e07995e

Comment (syphzur) [reply PhotoPlusBackend/photoplus/.idea/intellij-javadocs-4.0.1.xml]: thanks

File: PhotoPlusBackend/photoplus/.idea/intellij-javadocs-4.0.1.xml (hunk 2 above)
Comment (dsuchy) [reply PhotoPlusBackend/photoplus/.idea/intellij-javadocs-4.0.1.xml]: Very nice

**Review by syphzur — APPROVED** [review]

**Review by xPapla — APPROVED** [review]
dobra robota, ale nie kasuj angulara

**Review by dsuchy — APPROVED** [review]

**Pull request merged**